peps/pep-0736.rst

```diff
@@ -2,20 +2,19 @@ PEP: 736
 Title: Shorthand syntax for keyword arguments at invocation
 Author: Joshua Bambrick <jbambrick@google.com>,
         Chris Angelico <rosuav@gmail.com>
-Sponsor: Guido van Rossum <guido@python.org>
-Discussions-To: https://discuss.python.org/t/syntactic-sugar-to-encourage-use-of-named-arguments/36217
+Discussions-To: https://discuss.python.org/t/pep-736-shorthand-syntax-for-keyword-arguments-at-invocation/43432
 Status: Draft
 Type: Standards Track
 Created: 28-Nov-2023
 Python-Version: 3.13
-Post-History: 14-Oct-2023
+Post-History: 17-Jan-2024
```

Comment:
Please keep all the dates listed here, and link each to the relevant discussion.